webapp/src/workspace.ts

```diff
@@ -1504,11 +1508,16 @@ export function installByIdAsync(id: string) {
 export async function saveToCloudAsync(h: Header) {
     pxt.debug(`cloud save to ${h.name} (${h.id})`)
     checkHeaderSession(h);
+    const saveStart = U.nowSeconds()
```

Comment:
No StopWatch utility class?

Reply:
There probably is, but it doesn't seem simpler than this?